tools/cli/commands.js

```diff
@@ -2512,6 +2514,7 @@ var runTestAppForPackages = async function (projectContext, options) {
     webArchs.push("web.cordova");
   }
   buildOptions.webArchs = filterWebArchs(webArchs, options['exclude-archs'], projectContext.appDirectory, options);
+  global.webArchs = buildOptions.webArchs;
```

Comment:
This is the simplest way to share webarch info within the Meteor run and test context, and later reuse it in other parts of the bundler to skip unnecessary compilation. I see that global is used in the bundler to share data, but this could also be done by passing parameters down to the isopack compiler (similar to the includeCordovaUnibuild params used there). Not sure which is better, but this is definitely faster and works as a prototype to evaluate how useful the approach is.

I might change it later or keep it and just add comments.